### PR TITLE
Update markup for Security page

### LINKED ISSUE #442: [redesign] Security Page - Update Markup
> If this is your first time tackling an issue that is part of the redesign project please read all the **Notes** section at the bottom of this issue before claiming it and starting to work on it

## To Do: 
- [ ] Update the markup for the `/security` page

--- 

### Notes (important) 

This issue is part of the [Ember Website Redesign RFC](https://github.com/emberjs/rfcs/blob/master/text/0425-website-redesign.md) and is one of the **last remaining issues** before we are ready to release. As we are moving very quickly on this project you should comment on this issue that you are taking it on so that we do not have multiple people working on the same issue. **Note:** please don't take on an issue if you do not think you can complete it within 24 hours 🙏 

This issue is **strictly to only update the markup** of a page on the website and should be possible to do using only components and class helpers in the new [ember-styleguide](https://deploy-preview-145--ember-styleguide.netlify.com/).

If you find that you need to create page-specific styles, complete the page as best you can and open a PR **without** adding any styles. Make a note in your PR what was missing from the styleguide that prevented you from completing the page.

When creating your PR you should target the `redesign-rfc` branch. This way when your PR is merged it will not update `master` it will instead update the long-lived branch that is tracking the website redesign work.

### PR DESCRIPTION
**Pull request by mswanson**
fixes #442

I wasn't sure if there was a mockup somewhere of what this was supposed to look like. I just followed the class usage in #449 as an example. I'm happy to update it if necessary.

### CODE REVIEW

File: app/templates/security.hbs [hunk 1]
```diff
@@ -1,75 +1,90 @@
...
-</p>
-
-<h2 id="toc_comments-on-this-policy">Comments on This Policy</h2>
-<p>
-  If you have any suggestions to improve this policy, please send an email to <a href="mailto:security@emberjs.com">security@emberjs.com</a>.
-</p>
+<div class="container">
+  <h1>Ember.js Security Policy</h1>
+  <section class="margin-top-medium" aria-labelledby="section-reporting-a-bug">
+    <h2>Reporting a Bug</h2>
```
Comment (MelSumner) [inline app/templates/security.hbs]: this is missing `id="section-reporting-a-bug"`

File: app/templates/security.hbs [hunk 2]
```diff
@@ -1,75 +1,90 @@
...
+      <li>Post on the <a href="http://discuss.emberjs.com/">Ember.js discussion forums</a> or ask in #emberjs.</li>
+    </ol>
+    <p>
+      Please note that the discussion forums and <a href="https://discordapp.com/invite/zT3asNS">emberjs discord server</a>
+      are public areas. When escalating in these venues, please do not discuss your issue. Simply say that you’re trying to
+      get a hold of someone from the security team.
+    </p>
+  </section>
+  <section class="margin-top-medium" aria-labelledby="section-disclosure-policy">
+    <h2>Disclosure Policy</h2>
```
Comment (MelSumner) [inline app/templates/security.hbs]: this h2 is missing `id="section-disclosure-policy"`

File: app/templates/security.hbs [hunk 3]
```diff
@@ -1,75 +1,90 @@
...
+      </li>
+      <li>
+        This process can take some time, especially when coordination is required with maintainers of other projects.
+        Every effort will be made to handle the bug in as timely a manner as possible, however it’s important that we
+        follow the release process above to ensure that the disclosure is handled in a consistent manner.
+      </li>
+    </ol>
+  </section>
+  <section class="margin-top-medium" aria-labelledby="section-receiving-security-updates">
+    <h2>Receiving Security Updates</h2>
```
Comment (MelSumner) [inline app/templates/security.hbs]: this h2 is missing `id="section-receiving-security-updates"`

File: app/templates/security.hbs [hunk 4]
```diff
@@ -1,75 +1,90 @@
...
+      is very low traffic, and it receives the public notifications the moment the embargo is lifted.
+    </p>
+    <p>
+      No one outside the core team and the initial reporter will be notified prior to the lifting of the embargo. We regret
+      that we cannot make exceptions to this policy for high traffic or important sites, as any disclosure beyond the
+      minimum required to coordinate a fix could cause an early leak of the vulnerability.
+    </p>
+  </section>
+  <section class="margin-top-medium" aria-labelledby="section-comments-on-this-policy">
+    <h2>Comments on This Policy</h2>
```
Comment (MelSumner) [inline app/templates/security.hbs]: this h2 is missing `id="section-comments-on-this-policy"`

**Comment by mswanson** [comment]
Is something amiss with the Travis integration? I just looked at the build for this PR and it shows passing on Travis but the PR still shows it as running?

**Pull request merged**